password_security/models/res_users.py

```diff
@@ -164,11 +164,11 @@ def _validate_pass_reset(self):
         :raises: UserError on invalidated pass reset attempt
         :return: True on allowed reset
         """
-        for rec_id in self:
-            pass_min = rec_id.company_id.password_minimum
+        for user in self:
+            pass_min = user.company_id.password_minimum
             if pass_min <= 0:
                 pass
```

Comment:
Shouldn't this be `continue`, not `pass`?

Reply:
Right!